### PR TITLE
Add Fuckermate

### PR DESCRIPTION
**Pull request by gimmeliina**

### CODE REVIEW

File: scrapers/Fuckermate.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      Tags:
+        Name:
+          selector: //div[@class="post-meta"]/a()
+      Performers: 
+        Name: //h1[@class="post-title"]/a()
+driver:
+  useCDP: true
+
+# Last Updated December 29, 2021
+
```
Comment (Belleyy) [inline scrapers/Fuckermate.yml:35]: Remove this empty line

**Comment by Belleyy** [comment]
You need to update `SCRAPERS-LIST.md`

File: scrapers/Fuckermate.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: Fuckermate
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fuckermate.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="post-header font-alt"][1]/h1()
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:10]: the selector gets multiple results
I am assuming you want this part ?
```yaml
      Title: //section[@id="video-section"]//div[@class="post-header font-alt"]/h1[1]
```

File: scrapers/Fuckermate.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      Details: 
+        selector: //*[@id="video-section"]/div/div/div/div/div[3]/div/div[1]/p[2]
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:12]: safer
```suggestion
        selector: //div[@class="post-entry"]//p[@class="text-justify"]
```

File: scrapers/Fuckermate.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      Image:
+        selector: //div[@class="vjs-poster"]/@style
+        postProcess:
+          - replace:
+              - regex: .+(?:url\(")(.+)(?:"\)\;)
+                with: $1
+      Date:
+        selector: //div[@class="post-meta"]/text()
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:20]: selector gets a lot of extras
```suggestion
        selector: //div[@id="video-frame"]/following-sibling::div//div[@class="post-meta"]/text()[1]
```

File: scrapers/Fuckermate.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        postProcess:
+          - parseDate: 2006-01-02 |
+      Studio:
+        Name:
+          fixed: FuckerMate
+      Tags:
+        Name:
+          selector: //div[@class="post-meta"]/a()
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:28]: this selector gets both the scene tags and the performer ones, was that the intention?
for plain scene tags
```yaml
          selector: //section[@id="video-section"]//div[@class="post-meta"]/a
```

File: scrapers/Fuckermate.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      Performers: 
+        Name: //h1[@class="post-title"]/a()
+driver:
+  useCDP: true
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:32]: not needed should be removed

File: scrapers/Fuckermate.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    scene:
+      Title: //div[@class="post-header font-alt"][1]/h1()
+      Details: 
+        selector: //*[@id="video-section"]/div/div/div/div/div[3]/div/div[1]/p[2]
+      Image:
+        selector: //div[@class="vjs-poster"]/@style
+        postProcess:
+          - replace:
+              - regex: .+(?:url\(")(.+)(?:"\)\;)
+                with: $1
```
Comment (bnkai) [inline scrapers/Fuckermate.yml:18]:
```suggestion
      Image: //videoplayer/@poster
```
this works and doesnt need the cdp

**Pull request merged**